Bangazon-Tinkr/Controllers/RubbishController.cs

```diff
@@ -41,6 +45,21 @@ public IActionResult GetAllRubbish()
 
             return Ok(allRubbish);
         }
+        // api/Rubbish/2/createRubbish
+        [HttpPost("{userId}/createRubbish")]
```

Comment:
- I think the addition of the userId and the /createRubbish are not necessary for the route.  You are already completing a Post request on the Rubbish controller so the route is different than the posts for through the other controllers.